src/3D/imgui.ini

```diff
@@ -0,0 +1,5 @@
+[Window][Debug##Default]
+Pos=60,60
+Size=400,400
+Collapsed=0
+
```

Comment:
Why this file?

Reply:
hmm I've no idea, something in the build generated it, I'm not sure what it is..